helpers/updateMemberRank.js

```diff
@@ -40,13 +41,29 @@ async function updateMemberRank(memberDiscordId, discordClient) {
         return;
     }
 
-    memberData.currentCabbages = Object.values(
-        memberData.itemizedCabbages
-    ).reduce((acc, val) => acc + val, playerDetails.ehp + playerDetails.ehb);
+    // Calculate current cabbages
+    let cabbageCount =
```

Comment:
I would put the code from line 44-62 in a separate function (`convertProgressionToCabbages(...)`) that takes the `memberData` and `playerDetails` as an input and returns the amount of cabbages (`cabbageCount`)

Reply:
I was already planning to do something similar for #23 so it won't take long :)

Reply:
Agreed, I am pulling this out into a seperate function. For now I am leaving it in this file rather than putting it in a seperate module. @DisneyManBro feel free to change that when you work on #23 :)